recipe/migrations/cuda102_110_111_112_ppc64le.yaml

```
@@ -0,0 +1,56 @@
+migrator_ts: 1645421738
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  paused: true
+  override_cbc_keys:
+    - cuda_compiler_stub
+  operation: key_add
+  check_solvable: false
+  primary_key: cuda_compiler_version
+  wait_for_migrators:
+    - aarch64andppc64leaddition
+
```

Are we expecting removal of versions < 11.2?
```suggestion
c_compiler_version:            # [ppc64le]
  - 10                         # [ppc64le]
cxx_compiler_version:          # [ppc64le]
  - 10                         # [ppc64le]
fortran_compiler_version:      # [ppc64le]
  - 10                         # [ppc64le]

cuda_compiler_version:         # [ppc64le]
  - 11.2                       # [ppc64le]

cudnn:                  # [ppc64le]
  - 8                   # [ppc64le]

cdt_name:  # [linux]
  - cos7   # [ppc64le]

docker_image:                                           # [os.environ.get("BUILD_PLATFORM", "").startswith("linux")]
   - quay.io/condaforge/linux-anvil-ppc64le-cuda:11.2   # [ppc64le and os.environ.get("BUILD_PLATFORM") == "linux-ppc64le"]

```